.github/workflows/belt-hash.yml

```
@@ -31,12 +31,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: RustCrypto/actions/cargo-cache@master
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@master
         with:
-          profile: minimal
           toolchain: ${{ matrix.rust }}
           target: ${{ matrix.target }}
```

These need to be changed to `targets:`

```suggestion
          target: ${{ matrix.targets }}
```

---

Yeah, the documentation of `dtolnay/rust-toolchain` says so, but when you look into the [action.yml](https://github.com/dtolnay/rust-toolchain/blob/e12eda571dc9a5ee5d58eecf4738ec291c66f295/action.yml) it also mentions `target` as an alias for `targets`. So both variants should work, but I'll change it anyway.

---

Aah, interesting

---

I guess that alias was intentionally added for easier migration from `actions-rs/toolchain` where the parameter is named `target`.